modules/manage/partials/mountable-topics.adoc

```diff
@@ -133,28 +160,74 @@ curl -X POST http://localhost:9644/v1/topics/mount -d {
   "topics": [
     {
       "source_topic_reference": {"ns": "kafka", "topic": "<topic-1-name>/<cluster-1-uuid>/<initial-revision>"}, 
-      "alias": {"ns": "kafka", "topic": "<new-topic-1-name>"}
+      "alias": {"topic": "<new-topic-1-name>"}
     }, 
     {
       "source_topic_reference": {"ns": "kafka", "topic": "<topic-2-name>"}
+    }
+  ]
+}
+```
++
+* `ns` is the topic namespace. This field is optional and only `kafka` is supported.
+* You may have multiple topics with the same name that are available to mount from object storage. This can happen if you have unmounted topics with this name from different clusters. To uniquely identify a source topic, use `<topic-name>/<cluster-uuid>/<initial-revision>` as the topic reference.
```

Comment:
```suggestion
* You may have multiple topics with the same name that are available to mount from object storage. This can happen if you have unmounted topics using the same name in different clusters. To uniquely identify a source topic, use `<topic-name>/<cluster-uuid>/<initial-revision>` as the topic reference.
```